src/Microsoft.ApplicationInsights.AspNetCore/Logging/Implementation/ApplicationInsightsLoggerProvider.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Microsoft.ApplicationInsights.AspNetCore.Logging
+{
+    using Microsoft.Extensions.Logging;
+
+    internal class ApplicationInsightsLoggerProvider : ILoggerProvider
```

Comment:
Logger gest added into `ILoggerFactory` in `ApplicationInsightsInitializer` so there is no need for user to have access to classes to be able to log.